prime-router/metadata/hl7_mapping/common/note.yml

```diff
@@ -9,11 +9,26 @@ elements:
     hl7Spec: [ '%{hl7NotePath}/NTE(%{noteIndex})-2' ]
 
```

Comment:
should these be `noteDetails.author` or `noteDetails.authorReference`? It's a little tricky tracking down `noteDetail` but it seems to always point to Annotation which has an element for `authorReference`

Reply:
I also saw one of the FHIR files that `authorString` was an element on observation/note. How does that get created?